x-pack/plugins/actions/server/builtin_action_types/jira/schema.ts

```diff
@@ -37,7 +37,7 @@ export const ExecutorSubActionSchema = schema.oneOf([
 ]);
 
 export const ExecutorSubActionPushParamsSchema = schema.object({
-  savedObjectId: schema.string(),
+  savedObjectId: schema.nullable(schema.string()),
```

Comment:
I don't see a similar change to the resilient connector, here: https://github.com/elastic/kibana/blob/9f2d66c8bf0bd52584573fca945c02961789c009/x-pack/plugins/actions/server/builtin_action_types/resilient/schema.ts#L40